tests/cypress/tests-e2e/shareShortLink.cy.js

```diff
@@ -225,4 +225,64 @@ describe('Testing the share menu', () => {
             })
         }
     )
+    context(
```

Comment:
You are doing way to much testing in the `beforeEach` hook, I think it makes more sense to transform the whole thing into a single test case